README.md

```diff
@@ -502,6 +511,115 @@ server:start()
 
 [socket_ref]: https://www.tarantool.io/en/doc/latest/reference/reference_lua/socket/#socket-tcp-server
 
+## Roles
+
+New roles could be accessed from this project.
+
+### `roles.httpd`
+
+This role can be used only with Tarantool 3. It allows configuring one or more
+HTTP servers. Those servers could be reused by several other roles.
+
+Example of the configuration:
+
+```yaml
+roles_cfg:
+  roles.httpd:
+    default:
+    - listen: 8081
+    additional:
+    - listen: '127.0.0.1:8082'
+```
+
+Server address should be provided either as a URI or as a single port
+(in this case, `0.0.0.0` address is used).
+
+User can access every working HTTP server from the configuration by name,
+using `require('roles.httpd').get_server(name)` method.
+If the `name` argument is `nil`, the default server is returned
+(its name should be equal to constant
+`require('roles.httpd').DEFAULT_SERVER_NAME`, which is `"default"`).
+
+Let's look at the example of using this role. Consider a new role
+`roles/hello_world.lua`:
+```lua
+local M = {}
+local server = {}
+
+M.validate = function()  end
+
+M.apply = function(conf)
+    server = require('roles.httpd').get_server(conf.httpd).httpd
```

Comment:
> get_server().httpd

Why don't we return `httpd` itself? If I got it right, the `server` consists of `httpd` and `routes`, while `routes` is some internal `role.httpd` thing that is not expected to be used or changed by external roles.